### PR TITLE
multi namespace install docs

### PR DESCRIPTION
**Pull request by oraNod**
Docs for #104

Needs backport to 2.0.x

### CODE REVIEW

File: documentation/asciidoc/topics/proc_install_command_line.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ $ {oc_new_project} ${WATCH_NAMESPACE} <2>
 <1> Creates a project into which you install {ispn_operator}.
 <2> Creates a separate project for the {brandname} cluster that {ispn_operator} controls.
 +
+You can create multiple projects if you want the {ispn_operator} to control more than one {brandname} cluster.
```
Comment (Crumby) [inline documentation/asciidoc/topics/proc_install_command_line.adoc:24]: Operator can control more `{brandname}` clusters even in one namespace

Comment (oraNod) [reply documentation/asciidoc/topics/proc_install_command_line.adoc]: thanks @Crumby I've reverted the changes as discussed

File: documentation/asciidoc/topics/proc_install_command_line.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ As an alternative to installing {ispn_operator} through the **OperatorHub** on {
 
 .Procedure
 
-. Create projects for {ispn_operator} and {brandname} clusters.
+. Create a projects for {ispn_operator} and {brandname} clusters.
```
Comment (rigazilla) [inline documentation/asciidoc/topics/proc_install_command_line.adoc:11]: seems better the old version :)

Comment (oraNod) [reply documentation/asciidoc/topics/proc_install_command_line.adoc:11]: right you are. well spotted, @rigazilla

**Review by Crumby — APPROVED** [review]

**Comment by ryanemerson** [comment]
Thanks @oraNod

**Pull request merged**